functions/nodejs-apps.bats

```diff
@@ -9,7 +9,7 @@ setup_file() {
 
 @test "installation-Frontail_is_running" {
 run frontail_setup
-if [ "$status" -ne 0 ]; then echo "$output" >&3; fi
+#if [ "$status" -ne 0 ]; then echo "$output" >&3; fi
```

Comment:
I fixed this one in my upcoming changes, because when we test openHAB is not running it is a simple fix. Change this line in the frontail setup:
``` bash
dashboard_add_tile frontail
```
to:
``` bash
if ! running_in_docker; then
    dashboard_add_tile frontail
fi
```

Reply:
ok, done

Reply:
The output was because of that line in the code. That is why I said that. If you look at my Travis logs the output is now clean (and I fixed the homegear test after spending all day yesterday troubleshooting.).

Reply:
hmm I inserted that but still got the output. Maybe made a mistake, will retry.

Reply:
Homegear tests keep timing out, hanging in homegear_setup().
Took your code from fixes-2 repo but no change.

Reply:
Hmm, let me take a look.

Reply:
Is the rest ok? Then approve please, we can merge yours quickly after to contain any corrections if needed.

Reply:
Try again using my updated code, don't forget the changes that I made to the BATS files and the changes I made to the Dockerfiles. Also make sure you add the new homegear service file to the includes directory.